### PR TITLE
Feature/pickers

### PR DESCRIPTION
**Pull request by johnleider**

### CODE REVIEW

File: src/components/pickers/DatePicker.js [hunk 1]
```diff
@@ -60,7 +60,11 @@ export default {
   computed: {
     inputDate: {
       get () {
-        return new Date(this.value ? `${this.value} 12:00:00` : Date.now())
+        if (!this.value) return new Date()
+        if (this.value instanceof Date) return this.value
+        if (!isNaN(this.value) || this.value.indexOf(':') !== -1) return new Date(this.value)
```
Comment (nekosaur) [inline src/components/pickers/DatePicker.js]: Should this not be && on line 65?

Comment (johnleider) [inline src/components/pickers/DatePicker.js]: yes

**Comment by dohomi** [comment]
Very fast animation on scrolling the calendar
fast-calendar-animation

**Comment by dohomi** [comment]
Picker not working on Safari 10.1
safari-issue

**Comment by dohomi** [comment]
Following issues currently:```
* click on year inside year selection not working on any browser
* Firefox & Chrome no Date selection possible
* years in the year-selection seems 16px to far right
```

**Pull request merged**